test/e2e/save_test.go

```diff
@@ -121,7 +121,6 @@ var _ = Describe("Podman save", func() {
 
 	It("podman save remove signature", func() {
 		podmanTest.AddImageToRWStore(ALPINE)
-		SkipIfRootless("FIXME: Need get in rootless push sign")
```

Comment:
This still seems to be failing. I restarted the tests in case they're flakes, but I think they're not.

Reply:
Worked when I tested it locally.  Will look.